javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,16 @@
+var fullName = 'Bryan Valdovinos';
+console.log('value of fullName:', fullName);
+
+var isCool = true;
+console.log('value of isCool', isCool);
+
+var totalPets = 0;
+console.log('value of totalPets:', totalPets);
+
+var unknown = null;
+console.log('value of unknown:', unknown);
+console.log('typeof unknown:', typeof unknown);
+
+var never =
```

Comment:
This is not syntactically correct. You should do `var never;`

Reply:
Goott ittt! Just fixed, thank you sir!